ports/templates/ports/port-detail/summary.html

```diff
@@ -0,0 +1,89 @@
+{% load format_names %}
+    <br>
+    <div class="row">
+        <div class="col-lg-8">
+            <table class="table table-striped border border-top border-muted">
+                <tr>
+                    <th scope="row">Homepage</th>
+                    <td><a href="{{ port.homepage }}" target="_blank">{{ port.homepage }}</a></td>
+                </tr>
+                <tr>
+                    <th scope="row">Version:</th>
+                    <td>{{ port.version }}</td>
+                </tr>
+                <tr>
+                    <th scope="row">Maintained By:</th>
+                    <td>{% if maintainers.all|length > 0 %}
+                        {% for maintainer in maintainers %}
+                                {% if maintainer.github %}
+                                    <a href="/maintainer/github/{{ maintainer.github }}">{{ maintainer.github }}</a>,
+                                {% else %}
+                                    <a href="/maintainer/email/{{ maintainer.name }}__{{ maintainer.github }}">{{ maintainer.github }}</a>,
+                                {% endif %}
+                        {% endfor %}
+                    {% else %}
+                        No Maintainers Found
+                    {% endif %}
+
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row">ClosedMaintainer:</th>
+                    <td>{{ port.closedmaintainer }}</td>
+                </tr>
+                <tr>
+                    <th scope="row">Variants:</th>
+                    <td>
+                        {% if variants.count > 0 %}
+                            {% for variant in variants %}
+                                <a href="{% url 'variant_list' variant.variant %}">{{ variant.variant }}</a>,
+                            {% endfor %}
+                        {% else %}
+                            No Information Found for Variants
+                        {% endif %}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row">License:</th>
+                    <td>{{ port.license }}</td>
+                </tr>
+                <tr>
+                    <th scope="row">Platforms:</th>
+                    <td>{{ port.platforms }}</td>
+                </tr>
+                <tr>
+                    <th scope="row">Categories:</th>
+                    <td>{% for cat in port.categories.all %}
```

Comment:
same here. would look better if `{% for ...` starts in the next line after `<td>`.